app/javascript/stylesheets/lesson-content.css

```diff
@@ -79,7 +79,11 @@
   }
 
   .lesson-content a[href^="http"] {
-    @apply hover:no-underline;
+    @apply no-underline hover:underline;
```

Comment:
Whats the reason for this change?

Reply:
Threefold:
- This is how we tackle it everywhere else within the TOP websites; its only the in lesson links that diverge
- The new link colors in my experience tend to feel a bit bolder, and especially in areas where a lot of links where clustered together it occasionally started feeling cramped; by only showing the underline on hover we give everything a bit more space
- It just feels (at least to me) clearer if something gets more prominent on hover than not

Reply:
Thanks for laying it out. I think it was just a little bit jarring for me at first glance. Too used to looking at how it was I suppose lol. But after actually reading through some of the content, yep this is better, good call 💪